target/linux/ramips/image/mt76x8.mk

```diff
@@ -41,6 +41,13 @@ define Device/hc5661a
 endef
 TARGET_DEVICES += hc5661a
 
+define Device/hc5611
+  DTS := HC5611
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
```

Comment:
ralink_default_fw_size_16M is 16121856 bytes (0xf60000) but your factory partition has a size of 0xf70000. It has to be:

```
IMAGE_SIZE := 15808k
```